<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ParametersMaximizers</class>
 <widget class="QWidget" name="ParametersMaximizers">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>158</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>40</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Type</string>
   </property>
  </widget>
  <widget class="QComboBox" name="maximizeType">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Distance metric for the evaluation of NN</string>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <item>
    <property name="text">
     <string>Random Search</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Random Walk</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>PoWER</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Gradient Descent</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Donut</string>
    </property>
   </item>
  </widget>
  <widget class="QDoubleSpinBox" name="varianceSpin">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="minimum">
    <double>0.010000000000000</double>
   </property>
   <property name="maximum">
    <double>1.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.050000000000000</double>
   </property>
   <property name="value">
    <double>0.100000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="varianceLabel">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Search Variance</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="adaptiveCheck">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>70</y>
     <width>70</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Adaptive</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="kSpin">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>100</y>
     <width>60</width>
     <height>25</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>Number of iterations to perform
0: infinite</string>
   </property>
   <property name="minimum">
    <number>1</number>
   </property>
   <property name="maximum">
    <number>99999999</number>
   </property>
   <property name="value">
    <number>10</number>
   </property>
  </widget>
  <widget class="QLabel" name="countLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>102</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="text">
    <string>K</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
